### DISCUSSION TITLE
Nested Lists

### DISCUSSION THREAD

**Discussion by dolfbarr** [community/community#9222] [Other Feature Feedback, Questions, & Ideas]
A feature to make nested lists.

**Comment by testfailed** [comment]
It might sounds like too much but I'm already structuring lists like

/my
/my/participating
/my/private
/my/stacks
/my/toy-project
/ref
/ref/awesome
/ref/guide/{oss,style}
/ref/web/{backend,frontend}
/(project-a-related)
/(project-b-related)
/etc
/etc/dotenv
/etc/fun
/etc/inspiring
...

and directory-like structuring feature would be very helpful to me. (Although original intension might not meant to be too complex like it)

\+ Increasing maximum limit of lists would be greatly appreciated as well!
